internal/store/builder.go

```diff
@@ -96,12 +96,12 @@ func (b *Builder) WithEnabledResources(r []string) error {
 		}
 	}
 
-	var copy []string
-	copy = append(copy, r...)
+	var c []string
```

Comment:
why are you renaming `copy` to a single character?

Reply:
Renamed, it originally collided with the [`copy`](https://pkg.go.dev/builtin#copy)  function in the `builtin` package.